ui/ducks/metamask/metamask.js

```diff
@@ -307,7 +307,7 @@ export function getConversionRate(state) {
 }
 
 export function getNativeCurrency(state) {
-  return state.metamask.nativeCurrency;
+  return state.metamask.provider.ticker;
```

Comment:
We should not do this when the currencyratecontroller is being used (i.e. the toggle is on). Because it could make the conversion rate and the native currency be out of sync.